README.md

```diff
@@ -31,6 +31,20 @@ You can also set this GitHub config via an `.env` variable, by passing `GITCLERK
 
 ## Features
 
+### Viewing Mode
+
+Git clerks allow users to set their main `viewingMode` as `file-browser` or `session`
+
+```js
+globalThis.gitClerkConfig = {
+  [...]
+  viewingMode: "file-browser", // "file-browser" or "session"
```

Comment:
I am unsure about the `viewingMode` property name - this is really only for the initial entry point, as even when starting with `session` the user can go back and forth to the "temporary" file browser.
Maybe call it something like `initialView`? And initial view could be "file-browser" or "session**s**"?

Reply:
Another option would be to really have a fixed "viewingMode", so if it is `file-browser`, don't display the "temporary" file-browser drawer and really show the /file-browser view as main view even after navigation; following the logic the "session list/session" view would be in a temporary drawer. Currently it is a bit confusing, as the user starts in viewing mode "file-browser" with the header "File Browser" and the only button "View Sessions":
![image](https://github.com/user-attachments/assets/568d3fe5-6309-4978-a8db-a233ee641700)

Then, the user clicks on "View Sessions", then again on "Browse Files", but now the header looks completely different:
![image](https://github.com/user-attachments/assets/c3f2e0e0-91fb-4fa5-b424-b4863cc15010)